frontend/src/ClientDataForm.js

```diff
@@ -125,12 +161,20 @@ const ClientDataForm = () => {
           <h1>Client Form</h1>
         </div>
         <form method="post" onSubmit={log_information}>
-          {questions.map((question, index) => renderQuestion(question, index))}
+          {questions
+            .filter((question) => question.language_fk_id === Number(language))
+            .map((question, index) => renderQuestion(question, index))}
           <div className="row my-2">
             <div className="col mb-2" align="center">
-              <button className="btn btn-primary btn-lg" type="submit">
-                Submit
-              </button>
+              <Link to="/selectlanguage"> 
```

Comment:
This causes the button to skip the submit function, so it just links to `/selectlanguage` without calling `log_information`. This means no POST is made and no submission message pops up.